storage_scrubber/src/main.rs

```diff
@@ -380,6 +380,7 @@ pub async fn scan_pageserver_metadata_cmd(
 
             if summary.is_fatal() {
                 tracing::error!("Fatal scrub errors detected");
+                std::process::exit(1);
```

Comment:
can we make it a command line parameter? (i.e., `scrubber --exit-code` returns 1 when there's fatal error, and without the parameter, it returns 0.

I saw there are "fatal errors" across all regions and I don't want the alerts to fire now. Once we resolved all the fatal errors, I'll add such parameters to the infra repo and the alert will start to fire.

Reply:
Yes, added in f4e9a797ebe81f297beac9aacea2d15b61153f4b. and I think once we resolve all the fatal errors, it will make more sense to post scrub results to storcon (with `--post`).